ripple/target/appengine-staging/__static__/auth.js

```diff
@@ -0,0 +1,133 @@
+// function to sign up with Google
+function signUpWithGoogle() {
+  var provider = new firebase.auth.GoogleAuthProvider();
+  provider.addScope('profile');
+  auth.signInWithPopup(provider).then(function(result) {
+    // window.location = 'landingbusiness.html';
+    console.log("Success: Google account linked");
+    var token = result.credential.accessToken;
+    var user = result.user;
+    var name = user.displayName;
+    var uid = user.uid;
+    var email = user.email;
+    var isbusinessOwner = false;
+    console.log("name: " + name);
+    console.log("uid: " + uid);
+    sendEmailVerification(uid);
+    addNewUser(uid, name, isBusinessOwner, email);
+  }).catch(function(error) {
+    console.log(error);
+    // Handle Errors here.
+    var errorCode = error.code;
+    var errorMessage = error.message;
+    var email = error.email;
+    var credential = error.credential;
+    // ...
+  });
+}
+
+function addNewUser(uid, name, isBusinessOwner, email) {
+  console.log("addNewUser("+ uid + ", " + name + ", " + isBusinessOwner + ")");
+  db.collection("users").doc(uid).set({
+    uid: uid,
+    name: name,
+    isBusinessOwner: isBusinessOwner,
+    email: email,
+    // blobKey for default avatar photo
+    blobKey: "XEkBWMfbItUDt70PromXrQ",
+  })
+  .then(function(docRef) {
+    console.log("Document successfully written!");
+  })
+  .catch(function(error) {
+    console.error("Error adding document: ", error);
+  });
+}
+
+// function to sign in with Google
+function signInWithGoogle() {
+  // Signed in state will persist until user logs out 
+  firebase.auth().setPersistence(firebase.auth.Auth.Persistence.LOCAL)
+    .then(function() {
+      var provider = new firebase.auth.GoogleAuthProvider();
+      auth.signInWithPopup(provider).then(function(result) {
+        // window.location = 'landingbusiness.html';
```

Comment:
nit:remove

Reply:
I have uncommented the line! Fixed.

Reply:
In my src auth.js file, the line is uncommented. I believe this target file should automatically update on its own.